### PR TITLE
Document React SDK packages

### PR DESCRIPTION
**Pull request by alexandratran**
Document how-to instructions for `@metamask/sdk-react` and `@metamask/sdk-react-ui`.

Preview:
- https://docs.metamask.io/react-sdk/wallet/how-to/connect/set-up-sdk/javascript/react/
- https://docs.metamask.io/react-sdk/wallet/how-to/connect/set-up-sdk/javascript/react/react-ui/

### CODE REVIEW

File: wallet/how-to/connect/set-up-sdk/javascript/react/index.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+### 2. Import the SDK
+
+In your project script, add the following to import the SDK:
+
+```javascript
+import { MetaMaskProvider } from '@metamask/sdk-react';
+```
+
+### 3. Wrap your project with MetaMaskProvider
```
Comment (bgravenorst) [inline wallet/how-to/connect/set-up-sdk/javascript/react/index.md:47]: Can we format `MetaMaskProvider`? You may have mentioned there's an issue with using formatting in headings.

File: wallet/how-to/connect/set-up-sdk/javascript/react/react-ui.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+### 2. Import the SDK
+
+In your project script, add the following to import the SDK:
+
+```javascript
+import { MetaMaskUIProvider } from '@metamask/sdk-react-ui';
+```
+
+### 3. Wrap your project with MetaMaskUIProvider
```
Comment (bgravenorst) [inline wallet/how-to/connect/set-up-sdk/javascript/react/react-ui.md:49]: Same regarding formatting

File: wallet/how-to/connect/set-up-sdk/javascript/react/react-ui.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+For the full list of options you can set for `sdkOptions`, see the
+[JavaScript SDK options reference](../../../../../reference/sdk-js-options.md).
+
+### 4. Use the SDK
+
+Use the SDK by using the `useSDK` hook in your React components.
+See the [instructions for `@metamask/sdk-react`](index.md#4-use-the-sdk).
+
+### 5. Use the MetaMaskButton Component
```
Comment (bgravenorst) [inline wallet/how-to/connect/set-up-sdk/javascript/react/react-ui.md:85]:
```suggestion
### 5. Use the MetaMaskButton component
```

Comment (bgravenorst) [reply wallet/how-to/connect/set-up-sdk/javascript/react/react-ui.md:85]: Also see previous regarding formatting

**Review by bgravenorst — APPROVED** [review]
Minor feedback which may not apply

**Review by andreahaku — APPROVED** [review]
LGTM

**Pull request merged**